IntegrationTests/tests_02_syscall_wrappers/test_02_unacceptable_errnos.sh

```diff
@@ -47,10 +47,12 @@ if pipeErr != 0 {
 }
 let makeEBADFHappen =  CommandLine.arguments.dropFirst().first == .some("EBADF")
 let makeEFAULTHappen = CommandLine.arguments.dropFirst().first == .some("EFAULT")
-var whatevs: UInt8 = 123
+let makeEINVALHappen = CommandLine.arguments.dropFirst().first == .some("EINVAL")
+var whatevs: UInt8 = makeEINVALHappen ? 3021 : 123
```

Comment:
This does not produce EINVAL on Posix.write. From the manual page:

```
[EINVAL]           The pointer associated with fildes is negative.
```

`3021` is not negative.

Reply:
It is, however, larger than `UInt8` can hold, so the compile fails. This is why the tests are failing.